src/module/config.mjs

```diff
@@ -173,6 +173,7 @@ preLocalize("healingTypes", { key: "label" });
  * @property {boolean} [targeted]
  * @property {number} [maxSources]
  * @property {number} [defaultSpeed]
+ * @property {boolean} [hud_only]
```

Comment:
We dont typically use underscores in properties, just `hud` or perhaps `sheet: false` would be better.